lib/error.js

```diff
@@ -0,0 +1,36 @@
+/** takes an error object and serializes it to a plan object */
+function serializedError (input) {
+  if (!(input instanceof Error)) {
+    const error = new Error('attempted to serialize a non-error object')
+    return serializedError(error)
+  }
+  return {
+    errorType: input?.constructor?.name,
```

Comment:
We were previously using `input.name`. Why did you add `.constructor`? Will this preserve the name if `serializedError` is called twice? I think this may effectively happen if we're using a redacting logger.

Example: `safeError(safeError(axiosError))`

Reply:
Good catch, I was working on this right before I saw your comment, I have a fix updated.